_posts/0000-01-02-latentheatofcode.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+This is a new Branch
```

Comment:
## Coffee is just bean soup
![Alt Text](https://media.giphy.com/media/vFKqnCdLPNOKc/giphy.gif)